workers/api-server/src/http/wallet.rs

```diff
@@ -69,6 +69,32 @@ async fn append_task_and_await(
     Ok(task_id)
 }
 
+/// Enqueue tasks to pay all fees for a wallet
+///
+/// Modifies the wallet in place to remove fees
+async fn pay_wallet_fees(wallet: &mut Wallet, state: &State) -> Result<(), ApiServerError> {
+    let wallet_id = wallet.wallet_id;
+    for (mint, balance) in wallet.balances.iter_mut() {
+        if balance.relayer_fee_balance > 0 {
+            // Pay relayer fee
+            let task = PayOfflineFeeTaskDescriptor::new_relayer_fee(wallet_id, mint.clone())
+                .expect("infallible");
```

Comment:
nit but why do these constructors return results

Reply:
Some tasks _are_ fallible; I wanted interface consistently but didn't want to go so far as to force it into a trait. We can remove it down the line if we like